samples/meta-passthrough-sample/geometry.gd

```diff
@@ -2,4 +2,7 @@ extends OpenXRFbPassthroughGeometry
 
 
 func _process(delta: float) -> void:
-	position.x = 3 * sin(Time.get_ticks_msec() / 1000.0)
+	if not "--xrsim-automated-tests" in OS.get_cmdline_user_args():
+		# When we're running tests via the XR Simulator, we don't want the text
+		# to be animated, which can lead to differences in the screenshots.
```

Comment:
Assuming this comment was missed when copied over from the composition layer sample. :)

Reply:
Ah, thanks! Updated